cryptofeed/backends/redis.py

```diff
@@ -113,6 +128,16 @@ def __init__(self, *args, snapshots_only=False, snapshot_interval=1000, **kwargs
         super().__init__(*args, **kwargs)
 
 
+class BookSnapshot(RedisKeyCallback, BackendBookCallback):
```

Comment:
Can you rename this something that reflects its Redis/Redis K-V only?

Reply:
Sorry, I don't get it but you can tell me a name that match for you to rename it.

Reply:
BookSnapshotRedis should be fine